docs/api.md

```diff
@@ -115,14 +115,14 @@ _Appears in:_
 | Field | Description |
 | --- | --- |
 | `startTime` _string_ | (Optional) Test Start time. |
-| `parallelism` _integer_ | (Optional) Number of pods running at the same time. Defaults to `1` (Minimum `1`) |
+| `parallelism` _integer_ | (Optional) Number of pods running at the same time. Defaults to `1` (Mininum `1`) |
```

Comment:
Minimum?

Reply:
thanks! I fixed the typo https://github.com/st-tech/gatling-operator/pull/59/commits/9d7aef2e04c2ed3a7406d080f686e0024171e3a4